### PR TITLE
[FIX] POS settings and index.rst

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_printer_network/models/pos_printers.py [hunk 1]
```diff
@@ -14,6 +14,14 @@ class PosConfig(models.Model):
...
+        ('usb_printer', 'USB Printer'),
+        ('network_printer', 'Network Printer')], "Printer Type",
+        default='usb_printer', required=True,
+        help="Select the printer type you want to use receipt printing")
+    receipt_network_printer_ip = fields.Char(default=False, string="Network Printer IP", help="The ip address of the network printer for receipt")
+
+    @api.onchange('receipt_printer_type')
+    def _onchange_receipt_printer_type(self):
+        if self.receipt_printer_type == "usb_printer":
+            self.receipt_network_printer_ip = False
```
Comment (yelizariev) [inline pos_printer_network/models/pos_printers.py]: You should not remove value.

File: pos_printer_network/static/src/js/network_printer.js [hunk 2]
```diff
@@ -130,7 +130,17 @@ odoo.define('pos_restaurant.network_printer', function (require) {
...
-            if (!this.pos.config.usb_printer_active) {
+            var exist_usb_printer = this.pos.printers.find(function(printer){
+                return !printer.config.network_printer;
+            });
+
+            if (exist_usb_printer) {
+                this.pos.usb_printer_active = true;
+            } else {
+                this.pos.usb_printer_active = false;
+            }
```
Comment (yelizariev) [inline pos_printer_network/static/src/js/network_printer.js]: this.pos.usb_printer_active = exist_usb_printer;

File: pos_printer_network/static/src/js/network_printer.js [hunk 3]
```diff
@@ -130,7 +130,17 @@ odoo.define('pos_restaurant.network_printer', function (require) {
...
+
+            if (!this.pos.usb_printer_active) {
```
Comment (yelizariev) [inline pos_printer_network/static/src/js/network_printer.js]: Seems that you connect to specific POS, while ``this.pos.usb_printer_active`` means *there is at least one USB printer (on all POSes)

Comment (GabbasovDinar) [inline pos_printer_network/static/src/js/network_printer.js]: this.pos.usb_printer_active = this.pos.printers.find(function(printer){
                return !printer.config.network_printer;
            });

current POS

**Comment by yelizariev** [comment]
pos_printer_network/static/src/js/network_printer.js:138: [W7903(javascript-lint), ] Unexpected negated condition. [Error/no-negated-condition]

**Pull request merged**